confs/fdroid.json

```diff
@@ -72,6 +72,13 @@
 			"path": "fdroid/admin",
 			"aliases": [],
 			"token": "privatetoken"
+		},
+		{
+			"name": "ci-images",
+			"prefix": "https://gitlab.com",
+			"path": "fdroid/ci-images",
+			"aliases": ["ci", "ci-images"],
```

Comment:
Don't repeat the name as an alias. This should make the program crash because of the collision.